client/src/components/Login.js

```diff
@@ -1,13 +1,57 @@
 import React from "react";
+import axios from "axios";
 
-const Login = () => {
+const Login = (props) => {
   // make a post request to retrieve a token from the api
   // when you have handled the token, navigate to the BubblePage route
+  const [input, setInput] = React.useState({
+    username: 'Lambda School',
+    password: 'i<3Lambd4',
+  })
```

Comment:
For ease of development. 👍 

Just make sure to remove defaults like this when you're developing an app that's going to be published '' '